deployments/common/crds/k8s.nginx.org_policies.yaml

```diff
@@ -92,6 +92,8 @@ spec:
                   properties:
                     clientCertSecret:
                       type: string
+                    crl:
```

Comment:
I think we should use a more descriptive name than just `crl`.

Reply:
I think the crl part is fine, but maybe we could be more descriptive about it being a file name as opposed to the actual list? `crlFile` or `crlFileName` or something?

Reply:
I think this makes sense yeah. I'll go with `crlFileName` to be as explicit as possible.